web/themes/custom/usagov/templates/field--node--state-directory-record.html.twig

```diff
@@ -0,0 +1,88 @@
+{#
```

Comment:
I wonder whether we can use something like {% include directory ~ 'field--node--directory-record.html.twig' %} instead of all this content. (I have not tried that before.)

If that works, I think it's a plus, but we want to guard against some future developer deleting the file that has all the content! I would create an "includes" subdirectory under templates, move the content to a file there, and then make both of the field--node--etc files include it. That should be a pretty good clue.